### PR TITLE
Dev lunch 2017/03/24 blog post

### PR DESCRIPTION
**Pull request by anthonyec**
I've put together some of the highlights from Friday's dev lunch with a little description for each link.

Also I like the idea of mentioning what we had for lunch in each of these dev lunch blog posts.

### CODE REVIEW

File: _data/authors.yml [hunk 1]
```diff
@@ -6,4 +6,8 @@ chris:
 davidnormo:
     name: David Normington
     email: david.normington@signal-noise.co.uk
-    twitter: http://twitter.com/davidnormo
\ No newline at end of file
+    twitter: http://twitter.com/davidnormo
+anthony:
```
Comment (iplog) [inline _data/authors.yml]: Hum... spacing... (yeah I know, I know...)

File: _posts/2017-03-24-dev-lunch.markdown [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+## [Component based SVG Icon System](http://varun.ca/icon-component/)
+
+There isn't really a standard way to define icons in web interfaces and each way comes with it's pros and cons. Icon fonts and inline SVG allow for easy customization via CSS but both can be unwieldy to manage. Where as straight up `<img>` tags streamline the markup at the sacrifice of CSS edibility.
+
+Varun Vachhar talks about his work flow of using [SVG icon systems](https://css-tricks.com/svg-sprites-use-better-icon-fonts/) within React. This is a little reminiscent of Brent Jackson's work with [React icons](http://jxnblk.com/react-icons/)
+
+
+## [Styled components](https://github.com/styled-components/styled-components)
+
+This library allows you take take advantage of JavaScript when creating styles while writing normal CSS code. It does this by using ES6's [template literals](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Template_literals).
```
Comment (chrisbrown-io) [inline _posts/2017-03-24-dev-lunch.markdown]: >take take

File: _posts/2017-03-24-dev-lunch.markdown [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+There isn't really a standard way to define icons in web interfaces and each way comes with it's pros and cons. Icon fonts and inline SVG allow for easy customization via CSS but both can be unwieldy to manage. Where as straight up `<img>` tags streamline the markup at the sacrifice of CSS edibility.
+
+Varun Vachhar talks about his work flow of using [SVG icon systems](https://css-tricks.com/svg-sprites-use-better-icon-fonts/) within React. This is a little reminiscent of Brent Jackson's work with [React icons](http://jxnblk.com/react-icons/)
+
+
+## [Styled components](https://github.com/styled-components/styled-components)
+
+This library allows you take take advantage of JavaScript when creating styles while writing normal CSS code. It does this by using ES6's [template literals](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Template_literals). 
+
+It's an interesting idea but currently lacks some support for syntax highlighting in Sublime Text.
```
Comment (chrisbrown-io) [inline _posts/2017-03-24-dev-lunch.markdown]: plus, difficult to work with, with detailed css transitions
<https://github.com/styled-components/styled-components/issues/134>

Comment (anthonyec) [inline _posts/2017-03-24-dev-lunch.markdown]: You think this makes sense?

> It's an interesting idea but currently lacks some support for syntax highlighting in Sublime Text and has a few issues dealing with [highly dynamic styles](https://github.com/styled-components/styled-components/issues/134), such as animations.

**Pull request merged**